macOS/DuckDuckGo/Menus/MainMenu.swift

```diff
@@ -693,7 +697,7 @@ final class MainMenu: NSMenu {
                 NSMenuItem(title: "Reset MakeDuckDuckYours user settings", action: #selector(AppDelegate.resetMakeDuckDuckGoYoursUserSettings))
                 NSMenuItem(title: "Experiment Install Date more than 5 days ago", action: #selector(AppDelegate.changePixelExperimentInstalledDateToLessMoreThan5DayAgo(_:)))
                 NSMenuItem(title: "Change Activation Date") {
-                    NSMenuItem(title: "Today", action: #selector(AppDelegate.changeInstallDateToToday), keyEquivalent: "N")
+                    NSMenuItem(title: "Today", action: #selector(AppDelegate.changeInstallDateToToday))
```

Comment:
ℹ️ I removed this because this was wrong and conflicting with other shorcuts (it is a debug menu item). It seems it was just a slip on a copy/paste situation, given that all the other items do not have a shortcut.